addons/web/static/src/js/control_panel/control_panel_model.js

```diff
@@ -628,6 +618,9 @@ odoo.define('web.ControlPanelModel', function (require) {
             this.favoriteFilters.forEach(irFilter => {
                 const favorite = this._irFilterToFavorite(irFilter);
                 this._createGroupOfFilters([favorite]);
+                if (this.activateDefaultFavorite && !this.defaultFavoriteId && favorite.isDefault) {
```

Comment:
normally there can be only one default favorite

Reply:
Done